### PR TITLE
1070001- Stylus Re-Org: home

### PR DESCRIPTION
**Pull request by stephaniehobson**
Separated the home.styl file into multiple parts, made rtl improvements, removed lines that were not in use, reduced specificity where possible.

### CODE REVIEW

**Comment by stephaniehobson** [comment]
Let's wait on this one until PR #3172 is merged.

**Comment by openjck** [comment]
I don't think the test failure is legit, but I can't find a way to re-run the test. Hmm...

**Comment by stephaniehobson** [comment]
Needs a rebase anyway.

**Comment by openjck** [comment]
As per the meeting today, let's do this before #3172.

**Comment by stephaniehobson** [comment]
@openjck Does that mean you think it's okay to merge?

**Comment by openjck** [comment]
Definitely eligible to be merged. I'll look at this today when I do my PR run. :smile:

**Comment by openjck** [comment]
Awesome! I'm super excited to have more modular styles for the homepage. :tada:

I'm going to go through and make some minor code style improvements based on the [CSS Style Guide](https://mozweb.readthedocs.org/en/latest/reference/css-style.html). Even though we tend not to be super strict about following it, there are some points I try to keep in mind.

Please don't kill me. :smile:

File: media/stylus/components/home/column-callout.styl [hunk 1]
```diff
@@ -0,0 +1,104 @@
+/*
+colorful callouts
+====================================================================== */
+$callout-icon-spacing = 25px;
+$callout-icon-width = 90px;
+
+.column-callout    {
```
Comment (openjck) [inline media/stylus/components/home/column-callout.styl]: Just a code style nitpick. Maybe just one space between the selector and the brace?

File: media/stylus/home.styl [hunk 2]
```diff
@@ -1,593 +1,31 @@
...
-    /*  Change open auth text in white */
-    .oauth-login-options {
-        color: $blue-background-text-color;
-    }
 
-    .search-wrap {
-        input {
-            set-placeholder-style(color, $blue-background-text-color);
-        }
-    }
```
Comment (openjck) [inline media/stylus/home.styl]: Just another code style nitpick. Extra space from lines 29-31.

**Comment by openjck** [comment]
Otherwise this looks fantastic. Woo! :hamster:

**Comment by stephaniehobson** [comment]
Elected to :hocho: the whitespace instead of @openjck :P 

Amended!

**Comment by openjck** [comment]
A good decision. I approve!

**Comment by openjck** [comment]
Can't believe I forgot about this in #3193. We can remove all styles related to the headline. We should rebase against the latest master as well so that the commit removing the styles comes after the commit removing the headline.

**Comment by stephaniehobson** [comment]
Rebased on the latest master. Not sure what to do about the heading changes since that PR is proving to be controversial.

**Comment by openjck** [comment]
> Rebased on the latest master. Not sure what to do about the heading changes since that PR is proving to be controversial.

Ah, my fault. I assumed #3193 was already merged. We don't need to handle that here, then.

**Comment by stephaniehobson** [comment]
Rebasing again...

It's all about the rebase.

**Comment by stephaniehobson** [comment]
Bah, problems with the search filters.

**Pull request merged**